frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -205,35 +205,43 @@ class AccountsSummary extends Component<Props, State> {
-                                          <path d="M21 15v4a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2v-4"></path>
-                                          <polyline points="7 10 12 15 17 10"></polyline>
-                                          <line x1="12" y1="15" x2="12" y2="3"></line>
-                                      </svg>
-                                  </a>
-                              )
-                        )
-                        }
+                            exported ? (
+                                <A href={exported} title={exported} className="flex flex-row flex-start flex-items-center">
+                                    <span>
+                                        <img src={checkIcon} style="margin-right: 5px !important;" />
+                                        <span>{t('account.openFile')}</span>
+                                    </span>
+                                </A>
+                            ) : (
+                                <a onClick={this.export} title={t('accountSummary.exportSummary')}>
+                                    <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="#699ec6" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round">
+                                        <path d="M21 15v4a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2v-4"></path>
+                                        <polyline points="7 10 12 15 17 10"></polyline>
+                                        <line x1="12" y1="15" x2="12" y2="3"></line>
+                                    </svg>
+                                </a>
+                            )
+                        )}
                     </Header>
                     <div className="innerContainer scrollableContainer">
                         <div className="content padded">
-                            {data && <Chart
-                                dataDaily={data.chartDataMissing ? undefined : data.chartDataDaily}
-                                dataHourly={data.chartDataMissing ? undefined : data.chartDataHourly}
-                                fiatUnit={data.chartFiat}
-                                total={data.chartTotal}
-                                isUpToDate={data.chartIsUpToDate} />}
+                            {data ? (
+                                <Chart
+                                    dataDaily={data.chartDataMissing ? undefined : data.chartDataDaily}
+                                    dataHourly={data.chartDataMissing ? undefined : data.chartDataHourly}
+                                    fiatUnit={data.chartFiat}
+                                    total={data.chartTotal}
+                                    isUpToDate={!data.chartDataMissing || data.chartIsUpToDate} />
+                            ) : (
+                                <p style="text-align: center;">loading chart…</p>
```

Comment:
Could you maybe just reword the 'gathering information' text and show that here instead? so we don't add too many i18n entries.

Reply:
done